### DISCUSSION TITLE
Any way to fix Solarized color schemes such that arguments are legible?

### DISCUSSION THREAD

**Discussion by ccnly** [microsoft/terminal#14553] [Q&A] [answered]
When using the Solarized Dark color scheme in PowerShell, arguments are assigned a color that's illegible.  Is there a way to modify the out-of-the-box color scheme such that this doesn't happen?  I'm unsure which terminal color is getting applied.

![image](https://user-images.githubusercontent.com/53479999/207622136-c0b2042f-d387-48d0-b118-65d5d835a45e.png)
![image](https://user-images.githubusercontent.com/53479999/207622548-6fbe58f4-5bd8-49da-b93f-3926ba7ac5a4.png)

**Comment by zadjii-msft** [comment, marked as answer]
To be totally honest - Solarized is a bad color scheme for terminals. 
* https://github.com/microsoft/terminal/issues/13383#issuecomment-1174875973
* https://github.com/microsoft/terminal/issues/10027#issuecomment-831900824
* But seriously, this thread has actually useful information: https://github.com/microsoft/terminal/issues/6997#issuecomment-817381347

If it were me, I'd just patch all the gray "bright" colors (`brightGreen`, `brightYellow`, `brightBlue`, `brightCyan`) to all be equal to their dark equivalents. Sure some colors would be exactly the same, but it would be better than the terrible grays that Solarized ships by default.

**Comment by totkeks** [comment]
Hey @USMA56795, I setup my own color settings in my powershell profile: https://github.com/totkeks/dotfiles/blob/main/dotfiles/Documents/PowerShell/Profile/SolarizedDarkColorTheme.ps1

Basically it's all about setting proper colors for PSReadLine and then everything is fine. The defaults just don't work with the mapping solarized uses for their colors.

And here is my windows terminal color theme just for reference (in case I changed a color, but I think I didn't)
```json
{
            "background": "#002B36",
            "black": "#073642",
            "blue": "#268BD2",
            "brightBlack": "#002B36",
            "brightBlue": "#839496",
            "brightCyan": "#93A1A1",
            "brightGreen": "#586E75",
            "brightPurple": "#6C71C4",
            "brightRed": "#CB4B16",
            "brightWhite": "#FDF6E3",
            "brightYellow": "#657B83",
            "cursorColor": "#FFFFFF",
            "cyan": "#2AA198",
            "foreground": "#839496",
            "green": "#859900",
            "name": "Solarized Dark",
            "purple": "#D33682",
            "red": "#DC322F",
            "selectionBackground": "#FFFFFF",
            "white": "#EEE8D5",
            "yellow": "#B58900"
        },
```

**Comment by DHowett** [comment]
In addition to all the other answers here, I'd love to see you kick the tires on our new feature that automatically bumps up the contrast ratio when we detect that it would be unreadable otherwise.

It's available in Preview 1.16, and looks like this (Profile > Appearance)

<img width="735" alt="image" src="https://user-images.githubusercontent.com/189190/208215211-646797b8-3d9f-4534-a40c-109dafd82101.png">

**Reply by camilleg** [reply]
I found it in settings > profiles > powershell > appearance > text > auto adjust lightness.  Beautiful!
